consumer/pdata/generated_log_test.go

```diff
@@ -134,7 +134,7 @@ func TestResourceLogsSlice_Append(t *testing.T) {
 	es.Append(&emptyVal)
 	assert.EqualValues(t, *(es.At(7)).orig, *emptyVal.orig)
 
-	emptyVal2:= NewResourceLogs()
```

Comment:
You probably don't want to touch generated files.

Reply:
The `make fmt` command changed it so I left it.  Those kind of make targets ideally should be noops on the master branch so this will make it so.

Reply:
Also CircleCI should be flagging this kind of stuff too.

Reply:
I think we should generally avoid touching generated files otherwise when regenerating it will be difficult to tell what is the result of regeneration and what is the result of postgeneration fix.

Reply:
@tigrannajaryan See https://github.com/open-telemetry/opentelemetry-collector/pull/1458.